app/build.gradle

```diff
@@ -17,12 +17,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 25
-    buildToolsVersion "25.0.2"
+    compileSdkVersion 27
+    buildToolsVersion '27.0.0'
```

Comment:
Remove. This is now implied by the gradle plugin

Reply:
It's also updated automatically so it's not bad to have it there if the plugin maintains it. Otherwise how do you quickly check what build tools you're using?

Reply:
Android Studio handles the build tools version for you as of Android Studio 3.0. It looks at the compileSdkVersion to determine the build tools to use. It will use the highest version of the build tools that you have locally downloaded. So there is no reason to need to check since it has been automated 😄

Reply:
As discussed offline, leaving it for visibility (and to generate a diff in git)